docs/UserGuide.md

```diff
@@ -58,129 +60,133 @@ Amigos is a desktop application to help tech-savvy university students manage th
-  of a friend, the friend will be returned. e.g `findfriend n/Jim` will return `Jimmy Tan` and `findfriend ttl/Dinner`
+* The search is case-insensitive. e.g. `john` will match `John`
+* The order of the keywords does not matter. e.g. Doe John will match John Doe
+* As long as the keyword matches a substring of the name / log title / tags
+  of a friend, the friend will be returned. e.g. `findfriend n/Jim` will return `Jimmy Tan` and `findfriend ttl/Dinner`
   will return friends with log titles containing the substring `Dinner`.
-* Friends matching at least one keyword will be returned. e.g `findfriend n/John ttl/John` will return `John Doe`, `John Tan`,
+* Friends matching at least one keyword will be returned. e.g. `findfriend n/John ttl/John` will return `John Doe`, `John Tan`,
    as well as all friends with log titles containing the substring `John`.
 
 **Examples**:
-* `findfriend n/John t/neighbour`
-* `findfriend n/John n/Emily n/Russell`
+* `findfriend n/John t/neighbour` returns all friends with 'John' in the name and 'neighbour' in any of the tags.
+* `findfriend n/John n/Emily n/Russell`returns all friends with 'John' or 'Emily' or 'Russel' in the name.
 
 
 ## Logs management
-Amigos provides functionality to manage logs, which are essentially detailed notes about a specific friend.
+_Amigos_ provides functionality to manage logs, which are essentially detailed notes about a specific friend.
 
-### Adding a log: `addlog` / `al`
+### Adding a log: `addlog` or `al`
 
-Adds a log to an existing friend at the specified `INDEX` in Amigos.
+Adds a log to an existing friend at the specified `INDEX` or with the specified `NAME` in _Amigos_.
 The `INDEX` refers to the index number shown in the displayed person's list.
 
 **Format**: `addlog INDEX ? n/NAME ttl/TITLE [d/DESCRIPTION]`
 
 * Exactly one of `INDEX` or the `NAME` fields is compulsory.
-* If the `TITLE` argument is provided, then the `DESCRIPTON` argument is optional.
+* The `DESCRIPTION` argument is optional.
+* Note that duplicate logs are not allowed, and two logs are considered "duplicates" if they have the same title (case-sensitive). 
+  The rationale for this is that users should be minimally able to tell logs apart at a glance without restrictions on content.
 
 **Examples**:
-* `addlog 1 ttl/has a pet named poki`
-* `addlog John ttl/recommended movies d/the martian, interstellar, three idiots`
+* `addlog 1 ttl/has a pet named poki` adds a log to the person at index 1 of the "Friends" tab with the title "has a pet named poki".
+* `addlog n/Benson Meier ttl/recommended movies d/the martian, interstellar, three idiots`
 
-### Editing a log: `editlog` / `el`
+### Editing a log: `editlog` or `el`
 
-Edits an existing log of an existing friend in Amigos.
+Edits an existing log of an existing friend in _Amigos_.
 
-**Format**: `editlog INDEX ? n/NAME id/LOG_INDEX [ttl/NEW_TITLE] [d/NEW_DESCRIPTION]`
+**Format**:
```

Comment:
hmm, why display like this

Reply:
Some comment in the previous round mentioned it's "confusing". Either case, I think this is fine.

Reply:
Hmm, but we can't do this for the other commands with multiple optional fields though, do you think it's better to standardize?

Reply:
This is specifically for exclusive or arguments. And yes, I think so!